app/css/NameTag.css

```diff
@@ -1,47 +1,57 @@
 /* NameTag.css */
+body {
+  font-size: 1rem; /* Base body text size; reflects user/system settings */
+}
+
+h1, .tab-title {
+  font-size: 2rem; /* Header text size relative to base size */
+  font-weight: bold;
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
```

Comment:
Similar issue there, should be integrated to globals.css

Reply:
Resolved